CRM/Mailing/BAO/Mailing.php

```diff
@@ -916,6 +950,10 @@ private function _getTokens($prop) {
    * @return void
    */
   public function getTestRecipients($testParams) {
+    $session = CRM_Core_Session::singleton();
+    $sender_id = $session->get('userID');
```

Comment:
@eileenmcnaughton : Sure, https://wiki.civicrm.org/confluence/display/CRMDOC/PHP+Code+and+Inline+Documentation